spec/en/reference.md

```diff
@@ -120,6 +148,7 @@ If an agency can associate supplementary ridership data with a specific rider, r
 |  Field Name | Required | Details |
 |  ------ | ------ | ------ |
 | rider_id | **Required** | The **rider_id** field contains the ID of a unique rider. The **rider_id** is dataset unique. |
+| agency_id | Optional | The **agency_id** field contains the ID of the agency associated with the unique rider. This value is referenced from the agency.txt file. Use this field when you are providing data from more than one agency.  |
```

Comment:
Create hyperlink back to GTFS agency.txt file where mentioned in the description.